apps/api-documenter/src/start.ts

```diff
@@ -15,4 +15,4 @@ console.log(os.EOL + colors.bold(`api-documenter ${myPackageVersion} `
 
 const parser: ApiDocumenterCommandLine = new ApiDocumenterCommandLine();
 
-parser.execute();
+parser.execute().catch(console.error);
```

Comment:
If the original design is intentionally eat up the promise, we should `.catch(noop)`.

Reply:
The docs say that ["The returned promise will never reject"](https://microsoft.github.io/web-build-tools/api/ts-command-line.commandlineparser.execute.html).

Your fix here seems fine, but it might be useful to add a code comment saying "CommandLineParser.execute() should never reject the promise" to explain why we're okay with showing an ugly stack dump here (which is what `console.error(new Error('Example'))` would do)

---
In reply to: [239000045](https://github.com/Microsoft/web-build-tools/pull/983#discussion_r239000045) [](ancestors = 239000045)